hca-api-stub/voa-rest.js

```diff
@@ -74,7 +74,7 @@ function attach(app) {
       connector: require('loopback-connector-soap'),
       remotingEnabled: true,
       wsdl: path.join(__dirname, './voa.wsdl'),
-      url: endpoint.esPreprod,
+      url: 'http://vaausesrapp803.aac.va.gov:7401/voa/voaSvc',
```

Comment:
Is this a permanent change or just a temporary fix? Why not also do `endpoint.c7401`?